components/org.wso2.carbon.identity.discovery/src/main/java/org/wso2/carbon/identity/discovery/OIDProviderConfigResponse.java

```diff
@@ -78,6 +78,7 @@ public class OIDProviderConfigResponse {
     private Boolean backchannelLogoutSessionSupported;
     private String[] codeChallengeMethodsSupported;
     private String deviceAuthorizationEndpoint;
+    private boolean isTlsClientCertificateBoundAccessTokensEnabled;
```

Comment:
Lets change "enabled" to "supported". let make the same change in the other relevant places
```suggestion
    private boolean isTlsClientCertificateBoundAccessTokensSupported;
```

Reply:
Fixed in 13e64465f45bb0e715a99279ff095cc6892e9066